cli/commands/init.go

```diff
@@ -90,50 +90,47 @@ type InitOptions struct {
 }
 
 func (o *InitOptions) Run() error {
-	var dir string
-	// prompt for init directory if it's not set already
-	if o.InitDir == "" {
-		p := promptui.Prompt{
-			Label:   "Name of project directory ",
-			Default: defaultDirectory,
-		}
-		r, err := p.Run()
+	var infoMsg string
+	if o.InitDir == "." {
+		dir, err := os.Getwd()
 		if err != nil {
-			return handlePromptError(err)
-		}
-		if strings.TrimSpace(r) != "" {
-			dir = r
-		} else {
-			dir = defaultDirectory
+			return errors.Wrap(err, "error getting present working directory")
 		}
-	} else {
-		dir = o.InitDir
-	}
-
-	if o.EC.ExecutionDirectory == "" {
 		o.EC.ExecutionDirectory = dir
+		infoMsg = fmt.Sprintf(`hasura project initialised. execute the following command to continue:
+  hasura console
+`)
 	} else {
-		o.EC.ExecutionDirectory = filepath.Join(o.EC.ExecutionDirectory, dir)
-	}
-
-	var infoMsg string
-
-	// create the execution directory
-	if _, err := os.Stat(o.EC.ExecutionDirectory); err == nil {
-		return errors.Errorf("directory '%s' already exist", o.EC.ExecutionDirectory)
-	}
-	err := os.MkdirAll(o.EC.ExecutionDirectory, os.ModePerm)
-	if err != nil {
-		return errors.Wrap(err, "error creating setup directories")
-	}
-	infoMsg = fmt.Sprintf(`directory created. execute the following commands to continue:
+		// prompt for init directory if it's not set already
+		if o.InitDir == "" {
+			p := promptui.Prompt{
+				Label:   "Name of project directory ",
+				Default: defaultDirectory,
+			}
+			r, err := p.Run()
+			if err != nil {
+				return handlePromptError(err)
+			}
+			if strings.TrimSpace(r) != "" {
+				o.InitDir = r
```

Comment:
@ShraddhaAg Can we also handle if the user enters `.` in the promptui

Reply:
Ah right. Missed it. Done!